src/Components/Subtitle/index.css

```diff
@@ -0,0 +1,6 @@
+.subtitle{
```

Comment:
This is not following `SUIT CSS` naming conventions, the component name should be capitalized.